docs2/site/docs/migrations/migration5.md

```diff
@@ -430,3 +449,33 @@ string s = writer.ToString();
 `SDLPrinter` is a highly optimized visitor for asynchronous non-blocking SDL output
 into provided `TextWriter`. In the majority of cases it does not allocate memory in
 the managed heap at all.
+
+### 21. `AutoRegisteringObjectGraphType` changes
+
+The protected method `GetRegisteredProperties` has been renamed to `GetRegisteredMembers`
+and now supports properties, methods and fields, although fields are not included
+with the default implementation. Override the method in a derived class to include fields.
+
+New for v5, methods are included by default.  To revert to v4 behavior, which does not
+include methods, create a derived class as follows:
+
+```csharp
+public class AutoRegisteringObjectGraphTypeWithoutMethods<T> : AutoRegisteringObjectGraphType<T>
+{
+    public AutoRegisteringObjectGraphTypeWithoutMethods() : base() { }
+    public AutoRegisteringObjectGraphTypeWithoutMethods(params Expression<Func<T, object?>>[]? excludedProperties) : base(excludedProperties) { }
+    protected override IEnumerable<MemberInfo> GetRegisteredMembers() => base.GetRegisteredMembers().Where(x => x is PropertyInfo);
+}
+```
+
+Register this class within your DI engine like this:
+
+```cs
+services.AddTransient(typeof(AutoRegisteringObjectGraphType<>), typeof(AutoRegisteringObjectGraphTypeWithoutMethods<>));
+```
+
+### 22. `AutoRegisteringInputObjectGraphType` changes
+
+The protected method `GetRegisteredProperties` has been renamed to `GetRegisteredMembers`
+and now supports returning both properties and fields, although fields are not included
```

Comment:
Why without methods?

Reply:
The next PR enables support for methods
- #2891 

Of course not for input objects -- only output objects.

Reply:
This is an input object.  How could it support methods?

Reply:
How does the method differ from the property in this context? The property is the same method, even two.

Reply:
Yes, but how would it work?

```cs
//for output types
class Test
{
    public int Value1 => 3;
    public int Value2() => 3;
}
//usage within field resolver
var result1 = test.Value1;
var result2 = test.Value2();
//both read the value -- so far so good

//for input types
class Test
{
    public int Value { get; set; }
    //what is the signature?
    public void Value2(int newValue); //like this?? why?? very uncommon for data input
}
//usage internally
test.Value1 = 3;
test.Value2(3); //like this?  seems quite abnormal for such a use case
```

It just doesn't seem to make any sense to support methods for input objects.

Reply:
I imagined it differently. Perhaps I completely misunderstood the goal.

Reply:
I want to understand. Context of using output types - resolvers. You give the opportunity to work with CLR classes and GraphQL.NET maps them into graphtypes. So if method or property returns `int value` then this `value` translated to field with type `Int`. Right? What is the context of input types translation? Arguments?

Reply:
OK. The discard my suggestion abot methods.